scripts/splunk-default-config

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# no metrics
+export OTEL_METRICS_EXPORTER=none
+
+# otlp traces exporter
+export OTEL_TRACES_EXPORTER="${OTEL_TRACES_EXPORTER:-otlp}"
+# otlp-grpc protocol
+export OTEL_EXPORTER_OTLP_TRACES_PROTOCOL="${OTEL_EXPORTER_OTLP_TRACES_PROTOCOL:-grpc}"
```

Comment:
BTW this does not apply to Python. Python supports `otlp_proto_grpc` and `otlp_proto_http` values for `OTEL_TRACES_EXPORTER` with `otlp` being an alias for `otlp_proto_grpc`. If python decides to change the default to OTLP HTTP, this wrapper will automatically switch to OTLP http for python so may be in case of Python, we want to explicitly set `OTEL_TRACES_EXPORTER` to `otlp_proto_grpc`. 

Not requesting the change in this PR. Just FYI.